packages/gestalt-codemods/next/__testfixtures__/box-replace-marginleft-marginright.output.js

```diff
@@ -0,0 +1,7 @@
+// @flow strict
+import React from 'react';
+import { Box } from 'gestalt';
+
+export default function TestBox() {
+  return <Box marginStart={4} marginEnd={4} />;
```

Comment:
I would include responsive ones. We don't have any case in Pinboard, but as most of these Codemods... we must for our  due diligence!